manual_scans/aws/aws-ns-subdomain.py

```diff
@@ -26,7 +26,9 @@ def route53(profile):
         i = 0
         for page_records in pages_records:
             record_sets = [
-                r for r in page_records["ResourceRecordSets"] if r["Type"] == "NS" and r["Name"] != hosted_zone["Name"]
+                r
+                for r in page_records["ResourceRecordSets"]
+                if r["Type"] == "NS" and r["Name"] != hosted_zone["Name"] and r["Name"][0] != "_"
```

Comment:
Given this is growing and currently has to be managed in 3 places it might be worth pulling this check out into a common function that takes in r and hosted_zone and returns the result of the if statement